crd-convert/src/test/java/io/strimzi/kafka/crd/convert/converter/KafkaConnectConverterTest.java

```diff
@@ -0,0 +1,107 @@
+                .withEffect("dbdb")
+                .build())
+            .endSpec()
+            .build();
+        converted = specableConverter().testConvertTo(converted, ApiVersion.V1BETA2);
+        Assertions.assertEquals("kafka.strimzi.io/v1beta2", converted.getApiVersion());
+        Assertions.assertNull(converted.getSpec().getTolerations());
+        Assertions.assertEquals(1, converted.getSpec().getTemplate().getPod().getTolerations().size());
+        Assertions.assertEquals("foo", converted.getSpec().getTemplate().getPod().getTolerations().get(0).getKey());
+        Assertions.assertEquals("dbdb", converted.getSpec().getTemplate().getPod().getTolerations().get(0).getEffect());
+    }
+
+    @Override
+    protected void convertTolerationsToV1beta1(String fromApiVersion) {
+        KafkaConnect converted = new KafkaConnectBuilder()
+            .withApiVersion(fromApiVersion)
+            .withNewSpec()
+            .withNewTemplate()
+            .withNewPod()
+            .withTolerations(new TolerationBuilder()
+                .withKey("foo")
+                .withEffect("dbdb")
+                .build())
+            .endPod()
+            .endTemplate()
+            .endSpec()
+            .build();
+        converted = specableConverter().testConvertTo(converted, ApiVersion.V1BETA2);
+        Assertions.assertEquals("kafka.strimzi.io/v1beta2", converted.getApiVersion());
+        Assertions.assertNull(converted.getSpec().getTolerations());
+        Assertions.assertEquals(1, converted.getSpec().getTemplate().getPod().getTolerations().size());
+        Assertions.assertEquals("foo", converted.getSpec().getTemplate().getPod().getTolerations().get(0).getKey());
+        Assertions.assertEquals("dbdb", converted.getSpec().getTemplate().getPod().getTolerations().get(0).getEffect());
+    }
+
+    @Override
+    protected void convertAffinityToV1beta2(String fromApiVersion) {
+        KafkaConnect converted = new KafkaConnectBuilder()
+            .withApiVersion(fromApiVersion)
+            .withNewSpec()
+            .withAffinity(new AffinityBuilder()
+                .withNewNodeAffinity()
+                .addNewPreferredDuringSchedulingIgnoredDuringExecution()
+                .withWeight(100)
+                .endPreferredDuringSchedulingIgnoredDuringExecution()
+                .endNodeAffinity()
+                .build())
+            .endSpec()
+            .build();
+        converted = specableConverter().testConvertTo(converted, ApiVersion.V1BETA2);
+        Assertions.assertEquals("kafka.strimzi.io/v1beta2", converted.getApiVersion());
+        Assertions.assertNull(converted.getSpec().getAffinity());
+        Assertions.assertNotNull(converted.getSpec().getTemplate().getPod().getAffinity());
+        Assertions.assertEquals(100, converted.getSpec().getTemplate().getPod().getAffinity().getNodeAffinity().getPreferredDuringSchedulingIgnoredDuringExecution().get(0).getWeight());
+    }
+
+    @Override
```

Comment:
This is called `convertAffinityToV1beta1 ` ... but it seems to convert to v1beta2?

Reply:
@alesj can you fix it?